### PR TITLE
Create shubhamds24.json

### PR DESCRIPTION
**Pull request by shubhamds24**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6445"></a>

### CODE REVIEW

File: data/shubhamds24 [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Shubham Das",
+  "type": "personal",
+  "boi": "Travelling is good for health"
```
Comment (jaydeepkhatri) [inline data/shubhamds24:4]: The please check the spelling for "bio".
Also the JSON is invalid for the same line. It should have a trailing comma.

Here is how it should be 👇
```json
"bio": "Travelling is good for health",
```

File: data/shubhamds24 [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "links": [
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/sshubham_012?igshid=YmMyMTA2M2Y=",
+      "icon": "Fainstagram"
```
Comment (jaydeepkhatri) [inline data/shubhamds24:9]: The Correct instagram icon name is "FaInstagram". It is case-sensitive.

File: data/shubhamds24 [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/shubhamds24/LinkFree/new/main/data",
+      "icon": "FaGitHub"
```
Comment (jaydeepkhatri) [inline data/shubhamds24:14]: The correct github icon name is "FaGithub". It is case-sensitive.

**Review by jaydeepkhatri — CHANGES_REQUESTED** [review]
Hi @shubhamds24,
Thank you for contributing to Linkfree.

I have found few changes for your PR.
Please update the file name to `data/shubhamds24.json` 

Thanks :)

**Comment by SaraJaoude** [comment]
Given the length of time this PR has been open I will fix via a temporary branch and merge into main

**Pull request merged**